src/networking.c

```diff
@@ -1579,13 +1586,52 @@ int writeToClient(client *c, int handler_installed) {
             return C_ERR;
         }
     }
+
     return C_OK;
 }
 
+/* Process clients that were previously throttled. */
+void processUnthrottledClients() {
```

Comment:
I'm struggling to understand the modularity issue that Jim brought up. To me these just seem like a type of blocked clients, similar to paused clients. This code is nearly identical ```processUnblockedClients();```, in both cases a client should not longer consume commands until it's unblocked.